dwave/system/composites/linear_ancilla.py

```diff
@@ -0,0 +1,213 @@
+# coding: utf-8
+# Copyright 2018 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""Embedding composite to implement linear fields as polarized ancilla qubits.
```

Comment:
```suggestion
"""Composite to emulate linear coefficients with ancilla qubits biased with flux-bias offsets.
```

Reply:
Agree, except the word "emulate". This is an _alternative_ physical implementation of the Hamiltonian's linear bias.

Reply:
I am now aware that the word emulate has been used before in the docs. But I still think this not exactly an emulation since there are many ways to implement the Quantum Hamiltonian on a QPU. And this method can be used in addition to the conventional h in order to extend the linear bias range and avoid objective compression

Reply:
The thinking behind this is that the canonical implementation of the `\sum Q_{ii]_x_i` term of the QUBO is through the magnetic field applied as `h_i` linear bias and that FBO can emulate that same functionality of `h_i`

Reply:
Added to the docs in [d8416ff](https://github.com/dwavesystems/dwave-system/pull/530/commits/d8416ff59d1c1d22b3d1f88ad061ede737c64d75)

Yes, the FBO trick would be an emulation since the flux bias is constant throughout the anneal. It allows you to match a given h at one point in the schedule.